zone/bot.cpp

```diff
@@ -10757,22 +10770,15 @@ void Bot::ListBotSpellSettings()
 		bot_owner->Message(
 			Chat::White,
 			fmt::format(
-				"Setting {} | Spell: {} ({}) Enabled: {}",
+				"Setting: {} Spell: {} State: {} Remove: {}",
 				setting_number,
-				spells[bs.first].name,
-				bs.first,
-				bs.second.is_enabled ? "Yes" : "No"
-			).c_str()
-		);
-
-		bot_owner->Message(
-			Chat::White,
-			fmt::format(
-				"Setting {} | Priority: {} Levels: {} Health: {}",
-				setting_number,
-				bs.second.priority,
-				GetLevelString(bs.second.min_level, bs.second.max_level),
-				GetHPString(bs.second.min_hp, bs.second.max_hp)
+				Saylink::Silent(fmt::format("^botspellidinfo {}", bs.first), spells[bs.first].name),
+				Saylink::Silent(
+					fmt::format("^spellsettingstoggle {} {}",
+					bs.first, bs.second.is_enabled ? "False" : "True"),
+					bs.second.is_enabled ? "Enabled" : "Disabled"
+				),
+				Saylink::Silent(fmt::format("^spellsettingsdelete {}", bs.first), "Yes")
```

Comment:
Would prefer saylinks are "Add", "Enable", "Disable", similar to this.
![image](https://user-images.githubusercontent.com/89047260/205417696-0d6dfa73-6004-4448-97a1-8d41a7b0cae6.png)